src/main/java/calculator/StringAddCalculator.java

```diff
@@ -0,0 +1,53 @@
+package calculator;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+
+    private static final String DEFAULT_DELIMITER = "[,:]";
+    private static final String CUSTOM_DELIMITER = "//(.)\n(.*)";
+
+    public static int splitAndSum(String text) {
+        if (isBlank(text)) {
+            return 0;
+        }
+
+        Matcher m = Pattern.compile(CUSTOM_DELIMITER).matcher(text);
+        if (m.find()) {
+            String customDelimiter = m.group(1);
+            String[] values= m.group(2).split(customDelimiter);
+            return sum(toInts(values));
+        }
+        String[] values = text.split(DEFAULT_DELIMITER);
+        return sum(toInts(values));
+    }
+
+    private static boolean isBlank(String text) {
+        return text == null || text.isBlank();
+    }
+
+    private static int sum(int[] numbers) {
+        int sum = 0;
+        for (int number : numbers) {
+            sum += number;
+        }
+        return sum;
+    }
+
+    private static int[] toInts(String[] values) {
```

Comment:
```suggestion
    private static List<Integer> toInts(List<String> values) {
```

[이펙티브 자바 아이템 28](https://madplay.github.io/post/prefer-lists-to-arrays) 에서는 배열보다는 리스트를 사용하기를 권장합니다.
이번 기회에 이펙티브 아이템 28에 대해서 공부해보시고,
언제 배열을 쓰고 언제 리스트를 쓰면 좋을지 본인만의 기준을 코멘트로 남겨주시면 좋을 것 같습니다!